In [39]:
# ----------------------------------------------------------------
# User's Number Guess Program 
# ----------------------------------------------------------------
# Student   :   Abimbola Ayeni
# Supervisor:   Georgina Stanley
#               Technical Trainer
# Organization: Tech Talent Academy (TTA)
#               55, Colmore Row, 
#               Birmingham, B3 2AA
#            or Origin Workspace, 40 Berkeley Square, 
#               Bristol, BS8 1HP
# ----------------------------------------------------------------
#               Data Science: Phyton
#               Week 2 (Tuesday, 16.11.21, 4.30pm - 6.00pm)
#               Module 1
#               Task 1
# ----------------------------------------------------------------
# TTA: where Talent Meet Tech.
# ----------------------------------------------------------------
# print("Hi Dear")
# print("What is your gender")
UserGender = input("What is your gender? Male = 1 or Female = 2")
if UserGender == "Male":
#if UserGender == 1:
    print("Hello Handsome!")
elif UserGender == "Female":
#elif UserGender == 2:
    print("Heloo Pretty!")
if UserGender == "Male":
    print ("What is your name handsome?", "Trust you are good")
if UserGender == "Female":
    print ("What is your name pretty?", "Trust you are good")
UserName = input ("What is your name")
#print (UserName)
print (UserName, "You are most welcome to my website")
import random
GuessNumber = random.randint(1,10)
print("Great, "+ UserName + "#")
print("I want to reveal my secret to you." )
print("hmmm...")
print("Shhh!...my favourite number is between 1 and 10")
print("Can you guess right!")
UserGuessNo = input("Guess a number, please!")
if UserGuessNo == GuessNumber:
    print("Excellent, " + UserName + "! You guessed right")
    print("You are such a Genious")
    print("...but please don't tell anyone my favourite number")
    print("Bye, see you later")
elif UserGuessNo != GuessNumber:
    print("Almost right")
    print("Good luck next time")
    print("Bye, see you later")

Hello Handsome!
What is your name handsome! Trust you are good
bimb You are most welcome to my website
Great, bimb#
I want to reveal my secret to you.
hmmm...
Shhh!...my favourite number is between 1 and 10
Can you guess right!
Almost right
Good luck next time
Bye, see you later


----------------------------------------------------------------------
Week 2 (16.11.21): Module 1: Task 1 Questions
----------------------------------------------------------------------
Write a program that does the following:
a) Stores a random number (1-10) in a variable –see hint below.
b) Asks a user for their name and stores this in a variable.
c) Asks a user to guess the number between 1 and 10.
d) Tells the user whetherthey have guessed correctly.
----------------------------------------------------------------------
Output Sample of Task 1
----------------------------------------------------------------------
[Hello Handsome!
What is your name handsome! Trust you are good
bola You are most welcome to my website
Great, bola#
I wnat to reveal my secret to you.
hmmm...
Shhh!...my favourite number is between 1 and 10
Can you guess right!
Almost right
Good luck next time
Bye, see you later]
----------------------------------------------------------------------
In Number Guess programm, the user should guess between 1 and 10. 
Need further work.  No number I put between 1 and 10 was right. 

In [60]:
# ----------------------------------------------------------------
# User's Favourite Number Program
# ----------------------------------------------------------------
# Student   :   Abimbola Ayeni
# Supervisor:   Georgina Stanley
#               Technical Trainer
# Organization: Tech Talent Academy (TTA)
#               55, Colmore Row, 
#               Birmingham, B3 2AA
#            or Origin Workspace, 40 Berkeley Square, 
#               Bristol, BS8 1HP
# ----------------------------------------------------------------
#               Data Science: Phyton
#               Week 2 (Tuesday, 16.11.21, 4.30pm - 6.00pm)
#               Module 1
#               Task 2
# ----------------------------------------------------------------
# TTA: where Talent Meet Tech.
# ----------------------------------------------------------------
# print("Hi Dear")
# print("What is your gender")
UserGender = input("What is your gender? Male = 1 or Female = 2")
if UserGender == "Male":
#if UserGender == 1:
    print("Hello Handsome!")
elif UserGender == "Female":
#elif UserGender == 2:
    print("Heloo Pretty!")
if UserGender == "Male":
    print ("What is your name handsome?", "Trust you are good")
if UserGender == "Female":
    print ("What is your name pretty?", "Trust you are good")
UserName = input ("What is your name")
#print (UserName)
print (UserName, "You are most welcome to my website")
import random
GuessNumber = random.randint(1,100)
print("Great, "+ UserName + "#")
print("What is your favourite number between 1 and 100?" )
UserGuessNo = input("Guess a number, please!")
print("hmmm...")
print("Shhh!...your favourite number is, ",  UserGuessNo)
print("Am I right!")
UserResponse = input("YES OR NO!")
if UserResponse == "YES":
    print("Lovely, " + UserName + "! Your guess is as good as mine")
    print("Next...")
    print("Which animal loves to solves problems?")
    AnimalResponse = input("Which animal love to solves problems?")
    print(AnimalResponse)
    if AnimalResponse == "Pi-thon":
        print ("You are a genious")
    if AnimalResponse!= "Pi-thon":
        print ("Wwrroonngg!")
        print ("I have got my own problem to solve")
        print ("Anyway,the correct answer is Pi-thon")
        print ("Good luck next time")
        print ("Bye, see you soon")
else: 
    print ("Not exactly, " + UserName + "!")
    print ("Your number can not sit still at one place")
    print ("I am not good at guessing?")
    print("Next task")
    print("What type of snake is very good at mathematics?")
    AnimalResponse = input("What type of snake is very good at mathematics?")
    print(AnimalResponse)
    if AnimalResponse == "Adders":
        print ("You are a genious")
    if AnimalResponse!= "Adders":
        print ("too Wwrroonngg!")
        print ("I have got my own problem to solve")
        print ("Anyway,the correct answer is Adders")
        print ("Good luck next time")
        print ("Bye, see you soon")


Hello Handsome!
What is your name handsome? Trust you are good
gb You are most welcome to my website
Great, gb#
What is your favourite number between 1 and 100?
hmmm...
Shhh!...your favourite number is,  109
Am I right!
Not exactly, gb!
Your number can not sit still at one place
I am not good at guessing?
Next task
What type of snake is very good at mathematics?
cobra
too Wwrroonngg!
I have got my own problem to solve
Anyway,the correct answer is Adders
Good luck next time
Bye, see you soon


----------------------------------------------------------------------
Week 2 (16.11.21): Module 1: Task 2 Questions
----------------------------------------------------------------------
Write a program that asks a user for their favourite number between 1 and 100 and then tells them a joke based on the number. You should use a minimum of 3 jokes.
----------------------------------------------------------------------
Output Sample of Task 2
----------------------------------------------------------------------
[Hello Handsome!
What is your name handsome? Trust you are good
gb You are most welcome to my website
Great, gb#
What is your favourite number between 1 and 100?
hmmm...
Shhh!...your favourite number is,  109
Am I right!
Not exactly, gb!
Your number can not sit still at one place
I am not good at guessing?
Next task
What type of snake is very good at mathematics?
cobra
too Wwrroonngg!
I have got my own problem to solve
Anyway,the correct answer is Adders
Good luck next time]
----------------------------------------------------------------------
In Number Guess programm, the user was asked his or favourite number between 1 and 10. 
Need further work.   

In [68]:
# ----------------------------------------------------------------
# User's Favourite Food  
# ----------------------------------------------------------------
# Student   :   Abimbola Ayeni
# Supervisor:   Georgina Stanley
#               Technical Trainer
# Organization: Tech Talent Academy (TTA)
#               55, Colmore Row, 
#               Birmingham, B3 2AA
#            or Origin Workspace, 40 Berkeley Square, 
#               Bristol, BS8 1HP
# ----------------------------------------------------------------
#               Data Science: Phyton
#               Week 2 (Tuesday, 16.11.21, 4.30pm - 6.00pm)
#               Module 1
#               Task 3
# ----------------------------------------------------------------
# TTA: where Talent Meet Tech.
# ----------------------------------------------------------------
# print("Hi Dear")
# print("What is your gender")
UserGender = input("What is your gender? Male = 1 or Female = 2")
if UserGender == "Male":
#if UserGender == 1:
    print("Hello Handsome!")
elif UserGender == "Female":
#elif UserGender == 2:
    print("Heloo Pretty!")
if UserGender == "Male":
    print ("What is your name handsome?", "Trust you are good")
if UserGender == "Female":
    print ("What is your name pretty?", "Trust you are good")
UserName = input ("What is your name")
#print (UserName)
print (UserName, "You are most welcome to my website")
print("Great, "+ UserName + "#")
# .......................................................................
print("What is your favourite starter?" )
UserStarter = input("What is your favourite starter, please?")
print("My favourite starter is " + UserStarter)
# .......................................................................
print("What is your favourite main course?" )
UserMainCourse = input("What is your favourite main course, please?")
print("My favourite main course is " + UserMainCourse)
# .......................................................................
print("What is your favourite desert?" )
UserDesert = input("What is your favourite desert, please?")
print("My favourite desert is " + UserDesert)
# .......................................................................
print("What is your favourite drink?" )
UserDrink = input("What is your favourite drink, please?")
print("My favourite drink is " + UserDrink)
# .......................................................................
print ("Your favourite dinner strat with",UserStarter + " follow by", UserMainCourse + " then", UserDesert + " and step down with a cup of", UserDrink)
print ("Thank you for your patronage!")
print ("Bye! See you again soon.")


Kim You are most welcome to my website
Great, Kim#
What is your favourite starter?
My favourite starter is soup
What is your favourite main course?
My favourite main course is rice
What is your favourite desert?
My favourite desert is cake
What is your favourite drink?
My favourite drink is juice
Your favourite dinner strat with soup follow by rice then cake and step down with a cup of  juice
Thank you for your patronage!
Bye! See you again soon.


----------------------------------------------------------------------
Week 2 (16.11.21): Module 1: Task 3 Questions
----------------------------------------------------------------------
Write a program that allows user to enter their favourite starter, main course, dessert and drink.

Concatenate these and output a message which says –“Your favourite meal is  .........with a glass of....”
----------------------------------------------------------------------
Output Sample of Task 3
----------------------------------------------------------------------
[Kim You are most welcome to my website
Great, Kim#
What is your favourite starter?
My favourite starter is soup
What is your favourite main course?
My favourite main course is rice
What is your favourite desert?
My favourite desert is cake
What is your favourite drink?
My favourite drink is juice
Your favourite dinner strat with soup follow by rice then cake and step down with a cup of  juice
Thank you for your patronage!
Bye! See you again soon.]
----------------------------------------------------------------------
In this programm, the user was asked about his or favourite number meal. 
Need further work.   

In [83]:
# ----------------------------------------------------------------
# Depreciation Program  
# ----------------------------------------------------------------
# Student   :   Abimbola Ayeni
# Supervisor:   Georgina Stanley
#               Technical Trainer
# Organization: Tech Talent Academy (TTA)
#               55, Colmore Row, 
#               Birmingham, B3 2AA
#            or Origin Workspace, 40 Berkeley Square, 
#               Bristol, BS8 1HP
# ----------------------------------------------------------------
#               Data Science: Phyton
#               Week 2 (Tuesday, 16.11.21, 4.30pm - 6.00pm)
#               Module 1
#               Task 4
# ----------------------------------------------------------------
# TTA: where Talent Meet Tech.
# ----------------------------------------------------------------
# print("Hi Dear")
# print("What is your gender")
UserGender == input("What is your gender? Male = 1 or Female = 2")
if UserGender == "Male":
#if UserGender == 1:
    print("Hello Handsome!")
elif UserGender == "Female":
#elif UserGender == 2:
    print("Heloo Pretty!")
if UserGender == "Male":
    print ("What is your name handsome?", "Trust you are good")
if UserGender == "Female":
    print ("What is your name pretty?", "Trust you are good")
UserName = input ("What is your name")
#print (UserName)
print (UserName, "You are most welcome to my website")
print("Great, "+ UserName + "#")
# .......................................................................
#print("How much is the cost of your motorbike?" )
# UserBikeCost = input("How much is the cost of your motorbike, please?")
BikeCost = 2000
BikeDepRate = 10 
# .......................................................................
BikeDepYr1 = (BikeCost*BikeDepRate)/100
BikeCostYr1 = BikeCost - BikeDepYr1
print("Year 1: Motorbike Depreciation is ", + BikeDepYr1)
print("Year 1: Motorbike Costs is ", + BikeCostYr1)
# .......................................................................
if BikeCostYr1 < 1000:
    print ("Thank you, your Motorbike value is now ", BikeCostYr1)
elif BikeCostYr1 > 1000:
    BikeDepYr2 = (BikeCostYr1*BikeDepRate)/100
    BikeCostYr2 = BikeCostYr1 - BikeDepYr2
    print("Year 2: Motorbike Depreciation is ", + BikeDepYr2)
    print("Year 2: Motorbike Costs is ", + BikeCostYr2)
# .......................................................................
if  BikeCostYr2 < 1000:
    print ("Thank you, your Motorbike value is now", BikeCostYr2)
elif BikeCostYr1 > 1000:
    BikeDepYr3 = (BikeCostYr2*BikeDepRate)/100
    BikeCostYr3 = BikeCostYr2 - BikeDepYr2
    print("Year 3: Motorbike Depreciation is ", + BikeDepYr3)
    print("Year 3: Motorbike Costs is ", + BikeCostYr3)
# .......................................................................
if  BikeCostYr3 < 1000:
    print ("Thank you, your Motorbike value is now", BikeCostYr3)
elif BikeCostYr1 > 1000:
    BikeDepYr4 = (BikeCostYr3*BikeDepRate)/100
    BikeCostYr4 = BikeCostYr3 - BikeDepYr3
    print("Year 4: Motorbike Depreciation is ", + BikeDepYr4)
    print("Year 4: Motorbike Costs is ", + BikeCostYr4)
# .......................................................................
if  BikeCostYr4 < 1000:
    print ("Thank you, your Motorbike value is now",BikeCostYr4 )
elif BikeCostYr4 > 1000:
    BikeDepYr5 = (BikeCostYr4*BikeDepRate)/100
    BikeCostYr5 = BikeCostYr4 - BikeDepYr4
    print("Year 5: Motorbike Depreciation is ", + BikeDepYr5)
    print("Year 5: Motorbike Costs is ", + BikeCostYr5)
# .......................................................................
if  BikeCostYr5 < 1000:
    print ("Thank you, your Motorbike value is now", BikeCostYr5)
elif BikeCostYr5 > 1000:
    BikeDepYr6 = (BikeCostYr5*BikeDepRate)/100
    BikeCostYr6 = BikeCostYr5 - BikeDepYr5
    print("Year 6: Motorbike Depreciation is ", + BikeDepYr6)
    print("Year 6: Motorbike Costs is ", + BikeCostYr6)
# .......................................................................
if  BikeCostYr6 < 1000:
    print ("Thank you, your Motorbike value is now", BikeCostYr6)
elif BikeCostYr6 >= 1000:
    BikeDepYr7 = (BikeCostYr6*BikeDepRate)/100
    BikeCostYr7 = BikeCostYr6 - BikeDepYr6
    print("Year 7: Motorbike Depreciation is ", + BikeDepYr7)
    print("Year 7: Motorbike Costs is ", + BikeCostYr7)
# .......................................................................
if  BikeCostYr7 < 1000:
    print ("Thank you, your Motorbike value is now", BikeCostYr7)
    print ("Have a great day ahead!")
elif BikeCostYr7 >= 1000:
    BikeDepYr8 = (BikeCostYr7*BikeDepRate)/100
    BikeCostYr8 = BikeCostYr7 - BikeDepYr7
    print("Year 7: Motorbike Depreciation is ", + BikeDepYr8)
    print("Year 7: Motorbike Costs is ", + BikeCostYr8)
# .......................................................................

bim You are most welcome to my website
Great, bim#
Year 1: Motorbike Depreciation is  200.0
Year 1: Motorbike Costs is  1800.0
Year 2: Motorbike Depreciation is  180.0
Year 2: Motorbike Costs is  1620.0
Year 3: Motorbike Depreciation is  162.0
Year 3: Motorbike Costs is  1440.0
Year 4: Motorbike Depreciation is  144.0
Year 4: Motorbike Costs is  1278.0
Year 5: Motorbike Depreciation is  127.8
Year 5: Motorbike Costs is  1134.0
Year 6: Motorbike Depreciation is  113.4
Year 6: Motorbike Costs is  1006.2
Year 7: Motorbike Depreciation is  100.62
Year 7: Motorbike Costs is  892.8000000000001
Thank you, your Motorbike value is now 892.8000000000001
Have a great day ahead!


----------------------------------------------------------------------
Week 2 (16.11.21): Module 1: Task 4 Questions
----------------------------------------------------------------------
A motorbike costs £2000 and loses 10% of its value every year. Using a loop, print the value of the bike every following year until it falls below £1000.
----------------------------------------------------------------------
Output Sample of Task 4
----------------------------------------------------------------------
[Kim You are most welcome to my website
Great, bim#
Year 1: Motorbike Depreciation is  200.0
Year 1: Motorbike Costs is  1800.0
Year 2: Motorbike Depreciation is  180.0
Year 2: Motorbike Costs is  1620.0
Year 3: Motorbike Depreciation is  162.0
Year 3: Motorbike Costs is  1440.0
Year 4: Motorbike Depreciation is  144.0
Year 4: Motorbike Costs is  1278.0
Year 5: Motorbike Depreciation is  127.8
Year 5: Motorbike Costs is  1134.0
Year 6: Motorbike Depreciation is  113.4
Year 6: Motorbike Costs is  1006.2
Year 7: Motorbike Depreciation is  100.62
Year 7: Motorbike Costs is  892.8000000000001
Thank you, your Motorbike value is now 892.8000000000001
Have a great day ahead!]
----------------------------------------------------------------------
In this programm, the depreciation of motorbike 
Need further work, amateur iteration was done.   ]
----------------------------------------------------------------------

In [90]:
# ----------------------------------------------------------------
# Choice of Operator Program  
# ----------------------------------------------------------------
# Student   :   Abimbola Ayeni
# Supervisor:   Georgina Stanley
#               Technical Trainer
# Organization: Tech Talent Academy (TTA)
#               55, Colmore Row, 
#               Birmingham, B3 2AA
#            or Origin Workspace, 40 Berkeley Square, 
#               Bristol, BS8 1HP
# ----------------------------------------------------------------
#               Data Science: Phyton
#               Week 2 (Tuesday, 16.11.21, 4.30pm - 6.00pm)
#               Module 1
#               Task 5
# ----------------------------------------------------------------
# TTA: where Talent Meet Tech.
# ----------------------------------------------------------------
# print("Hi Dear")
# print("What is your gender")
UserGender == input("What is your gender? Male = 1 or Female = 2")
if UserGender == "Male":
#if UserGender == 1:
    print("Hello Handsome!")
elif UserGender == "Female":
#elif UserGender == 2:
    print("Heloo Pretty!")
if UserGender == "Male":
    print ("What is your name handsome?", "Trust you are good")
if UserGender == "Female":
    print ("What is your name pretty?", "Trust you are good")
UserName = input ("What is your name")
#print (UserName)
print (UserName, "You are most welcome to my website")
print("Great, "+ UserName + "#")
# .......................................................................
print ("You will be asked to input two numbers of your choice")
print ("input the first number of your choice")
FirstNumber = input("input the first number of your choice")
print("Your first number is ", FirstNumber)
print ("input the second number of your choice")
SecondNumber = input("input the second number of your choice")
print("Your second number is ", SecondNumber)
# .......................................................................
print ("You will be asked to input the type of mathematical operation you want to perform")
print ("input a for addition")
print ("input b for substraction")
print ("input c for Division")
print ("input d for Multiplication")
print ("input e for Square")
OperatorInput = input("input a for (+), b for (-), c for (/), d for (*) and e for (**)")
if OperatorInput == "a":
    print ("Please add")
    Add_a = (FirstNumber + SecondNumber)
    print("FirstNumber + SecondNumber")
    print ("The addition of your two numbers is ", Add_a)
# .......................................................................
elif OperatorInput == "b":
    print ("Please subtract")
    Sub_b = (FirstNumber - SecondNumber)
    print ("The addition of your two numbers is ", Sub_b)
# .......................................................................
elif OperatorInput == "c":
    print ("Please Divide")
    Div_c = FirstNumber / SecondNumber
    print ("The addition of your two numbers is ", Div_c)
# .......................................................................
elif OperatorInput == "d":
    print ("Please Multiply")
    Mult_d = FirstNumber * SecondNumber
    print ("The addition of your two numbers is ", Mult_d)
# .......................................................................
elif OperatorInput == "e":
    print ("Please Square")
    Squ_e = FirstNumber * SecondNumber
    print ("The addition of your two numbers is ", Squ_e)
# .......................................................................

lim You are most welcome to my website
Great, lim#
You will be asked to input two numbers of your choice
input the first number of your choice
Your first number is  20
input the second number of your choice
Your second number is  10
You will be asked to input the type of mathematical operation you want to perform
input a for addition
input b for substraction
input c for Division
input d for Multiplication
input e for Square
Please add
FirstNumber + SecondNumber
The addition of your two numbers is  2010
your addition final answer =  (20+10)


----------------------------------------------------------------------
Week 2 (16.11.21): Module 1: Task 5 Questions
----------------------------------------------------------------------
Write a program which will ask for two numbers from a user. Then offer a menu to the user giving them a choice of operator:

e.g. –Enter “a” if you want to add
            “b” if you want to subtract

Include +, -, /, *, ** square (to the power of). Once the user has selected which operator they wish to use, perform the calculation.
----------------------------------------------------------------------
Output Sample of Task 5
----------------------------------------------------------------------
[lim You are most welcome to my website
Great, lim#
You will be asked to input two numbers of your choice
input the first number of your choice
Your first number is  20
input the second number of your choice
Your second number is  10
You will be asked to input the type of mathematical operation you want to perform
input a for addition
input b for substraction
input c for Division
input d for Multiplication
input e for Square
Please add
FirstNumber + SecondNumber
The addition of your two numbers is  2010]
----------------------------------------------------------------------
In this programm, is suppose to handle mathematical operation.  Not getting the desired output yet, it is running but the string are not adding up.
Furtherwork needed
----------------------------------------------------------------------